sys/net/gnrc/network_layer/sixlowpan/frag/vrb/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_SIXLOWPAN_FRAG_VRB
+    bool "Configure GNRC 6LoWPAN Virtual reassembly buffer"
+    depends on MODULE_GNRC_SIXLOWPAN_FRAG_VRB
+    help
+        Configure GNRC 6LoWPAN Virtual reassembly buffer module using Kconfig.
+
+if KCONFIG_MODULE_GNRC_SIXLOWPAN_FRAG_VRB
+
+config GNRC_SIXLOWPAN_FRAG_VRB_SIZE
+    int "Size of the virtual reassembly buffer"
+    default 16
+    help
+        Only applicable with(@ref net_gnrc_sixlowpan_frag_vrb) module,
+        but has also a direct influence on the number of available
+        gnrc_sixlowpan_frag_rb_int_t entries.
```

Comment:
@miri64 is this still valid?

Reply:
Jain. The dependency is introduced in #11068. So rather it is not *yet* valid.

Reply:
Ok!